skimage/measure/_regionprops.py

```diff
@@ -209,9 +209,16 @@ def __init__(self, slice, label, label_image, intensity_image,
                  cache_active, *, extra_properties=None):
 
         if intensity_image is not None:
-            if not intensity_image.shape == label_image.shape:
-                raise ValueError('Label and intensity image must have the'
-                                 ' same shape.')
+            ndim = label_image.ndim
+            if not (
+                    intensity_image.shape[:ndim] == label_image.shape
+                    and intensity_image.ndim in [ndim, ndim + 1]
+                    ):
+                raise ValueError('Label and intensity image shapes must match,'
+                                 ' except for channels (last) axis.')
```

Comment:
```suggestion
                                 ' except for channel (last) dimension.')
```
Ok, but I think 'dimension' is more widely used throughout our documentation.

Reply:
@mkcor I prefer axis here...

Reply:
Ok; I suggested 'channel' as in 'noun as adjective.'

Reply:
Oops, missed that bit! :facepalm: Fixed!